File: VB/CustomEditorWinApp3/Form1.vb
Imports System.Windows.Forms

Namespace CustomEditor

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As System.EventArgs)
            categoriesTableAdapter.Fill(nwindDataSet.Categories)
            productsTableAdapter.Fill(nwindDataSet.Products)
        End Sub
    End Class
End Namespace

File: CS/CustomEditorWinApp3/Form1.cs
using System.Windows.Forms;

namespace CustomEditor
{
	public partial class Form1 : Form
	{
		public Form1()
		{
			InitializeComponent();
		}

		private void Form1_Load(object sender, System.EventArgs e)
		{
			this.categoriesTableAdapter.Fill(this.nwindDataSet.Categories);
			this.productsTableAdapter.Fill(this.nwindDataSet.Products);
		}
	}
}